api-reference/10 UI Components/dxForm/5 Item Types/GroupItem/captionTemplate.md

```diff
@@ -4,25 +4,120 @@ type: template
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a template that can be used to replace group caption with custom content.
 
 ##### param(data): Object
-<!-- Description goes here -->
+The group item's data.
 
 ##### field(data.caption): String
-<!-- Description goes here -->
+The group [caption](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#caption).
 
 ##### field(data.component): dxForm
-<!-- Description goes here -->
+The Form instance.
 
 ##### field(data.name): String
-<!-- Description goes here -->
+The group item's [name](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#name).
 
 ##### param(itemElement): DxElement
-<!-- Description goes here -->
+#include common-ref-elementparam with { element: "item" }
 
 ##### return: String | Element | jQuery
-<!-- Description goes here -->
+The template name or container.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property overrides the [caption](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/GroupItem/#caption).
+
+The following code example adds an icon to the existing caption:
```

Comment:
"the existing caption" - I would try to get rid of the article, because it seems like you're talking about a particular caption/element. I understand that you mean existing "content/text", but then you don't need an article. 

Use something uncountable, or plural - a generic description for this code. You can use it to add icons to "captions", or to "caption content". Obviously, you can also say "add an icon to a caption".

```suggestion
The following code example adds an icon to caption content:
```